regparser/layer/formatting.py

```diff
@@ -40,8 +40,14 @@ def build_header(xml_nodes):
     includes its colspan/rowspan"""
     stack = HeaderStack()
     stack.add(0, TableHeaderNode(None, 0))  # Root
+    ttitle_bump = 0
+    if len([_ for _ in xml_nodes if _.tag == "TTITLE"]):
```

Comment:
What do you think about this line instead? It seems a little more obvious to me what it's checking:
```python
if any(node.tag == "TITLE" for node in xml_nodes):
```